docs/repair_based_node_ops.md

```diff
@@ -0,0 +1,60 @@
+# Repair based node operations
+
+Here below is a simple introduction to the node operations Scylla supports and
+some of the issues.
+
```

Comment:
As soon as this is merged, this section becomes incorrect. So it should explain that it refers to the old method of streaming from one replica node operations.

Reply:
Fixed.